lib/concerns/listable.rb

```diff
@@ -0,0 +1,35 @@
+module Concerns
+  module Listable
+    # Listable class should have @@all class var available
```

Comment:
👍 - Good/necessary comment. 

It's not strictly that an `@@all` var is available though. It is necessary that an `.all` method is available.